partner_survey/views/res_partner_view.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<record id="act_partner_survey_input" model="ir.actions.act_window">
+    <field name="res_model">survey.user_input</field>
+    <field name="view_type">form</field>
+    <field name="name">Surveys</field>
+    <field name="view_mode">tree,form,calendar,graph</field>
```

Comment:
You're right. I missed that one. I have fixed it via direct commit with https://github.com/OCA/survey/commit/343e10fa3491a7e0adc1ee3810667648bd2bc33b